src/js/modules/sections/components/SectionPage.js

```diff
@@ -3,26 +3,121 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import injectSheet from 'react-jss';
-import sections from '../../sections';
+import { Grid, Row, Col } from 'react-bootstrap/lib/';
 
 import articles from '../../articles';
 
 const styles = {
   SectionPage: {
     color: '#000',
+  },
+  sectionName: {
+    width: '1080px',
+    height: '43px',
+    fontFamily: 'Canela',
+    fontSize: '36px',
+    fontWeight: '500',
+    textAlign: 'left',
+    color: '#000000',
+    padding: '0px 0px',
+    paddingLeft: '0px',
+    margin: '0px 0px',
+    marginBottom: '11px',
+  },
+  subsectionLink: {
+    listStyleType: 'none',
+    margin: '0px',
+    paddingLeft: '0px',
+    width: '1080px',
+    height: '7px',
+    fontFamily: 'Circular Std',
+    fontSize: '14px',
+    textAlign: 'left',
+    color: '#000000',
+    textTransform: 'uppercase',
+  },
+  articleList: {
+    width: '100%',
+    margin: '0px',
+  },
+  Horizontal: {
```

Comment:
In my review I already told her about naming. Don't review this until she goes through me.